NOTICE.txt

```diff
@@ -8,6 +8,327 @@ This document includes a list of open source components used in Mattermost Deskt
 
 --------
 
+## 7zip-bin
+
+This product contains '7zip-bin' by Vladimir Krivosheev.
+
+7-Zip precompiled binaries.
+
+* HOMEPAGE:
+  * https://github.com/develar/7zip-bin
+
+* LICENSE: MIT
+
+The MIT License (MIT)
+
+Copyright (c) 2016 Vladimir Krivosheev
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+---
+
+## @babel/core
```

Comment:
Are we sure we actually need to include all of these things that are used to build the web app but aren't actually included in it? Things like babel are only used to compile code for the app and ESLint is only used to check the styling. I don't think we've included those in the NOTICE.txt elsewhere since they aren't part of the finished product.

Reply:
Makes sense, I'm seeing now that some of those are dev dependencies which don't need to be included. I'll work on removing them. https://github.com/mattermost/desktop/blob/master/package.json#L41